tests/osaka/eip7692_eof_v1/eip7620_eof_create/test_eofcreate_failures.py

```diff
@@ -755,3 +755,60 @@ def test_eof_eofcreate_msg_depth(
         post=post,
         tx=tx,
     )
+
+
+def test_reentrant_eofcreate(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
+    """Verifies a reentrant EOFCREATE case, where EIP-161 prevents conflict via nonce bump."""
+    env = Environment()
+    contract_address = Address(0x1000000)
+    # Calls into the factory contract with 1 as input.
+    reenter_code = Op.MSTORE(0, 1) + Op.EXTCALL(address=contract_address, args_size=32)
```

Comment:
We could pass the contract address via calldata too in order to avoid hard-coding the address in the pre-alloc. This would allow us to run the test using execute command.

Reply:
Right! Thank you for flagging!